README.md

```diff
@@ -22,6 +22,9 @@ Portions Copyright (c) 2012-2013 The Cryptonote developers.
   - [Release staging schedule and protocol](#release-staging-schedule-and-protocol)
   - [Compiling Monero from source](#compiling-monero-from-source)
     - [Dependencies](#dependencies)
+  - [Internationalization](#Internationalization)
+  - [Using Tor](#Using Tor)
```

Comment:
This link is invalid -`#using-tor`. The others (`#internalization` and `#debugging`) are also all lower case, but I _think_ anchor tags are case insensitive. If you mouse-over next to the section name a link icon will appear with the valid anchor names.

Reply:
Apologies and thanks.
I'll correct in my branch.